packages/webapp/pages/onboarding.tsx

```diff
@@ -233,19 +255,103 @@ export function OnboardPage(): ReactElement {
-            : containerClass,
+          'flex justify-between w-full h-full flew-row px-6',
+          !isAuthenticating && !isFiltering && 'mt-20 w-full max-w-[75rem]',
         )}
       >
-        {getContent()}
+        <Logo
+          className={
+            !isAuthenticating && !isFiltering
+              ? 'w-auto'
+              : 'py-8 px-10 w-auto laptop:w-full'
+          }
+          logoClassName={
+            !isAuthenticating && !isFiltering ? 'h-logo-big' : 'h-logo'
+          }
+          position={LogoPosition.Relative}
+        />
+
+        {!isAuthenticating && !isFiltering && (
+          <span
+            className={classNames(
+              'flex items-center',
+              'text-theme-label-tertiary',
+            )}
+          >
+            <span className="hidden tablet:block">
+              Already a daily.dev member?
+            </span>
+            <Button
+              className="ml-3 btn-secondary"
+              onClick={() =>
+                setAuth({ isAuthenticating: true, isLoginFlow: true })
+              }
+            >
+              Log in
+            </Button>
+          </span>
+        )}
+      </header>
+      <div className="flex flex-wrap flex-1 justify-center px-6 mt-8 laptop:mt-20 w-full max-w-[75rem]">
+        {!isFiltering && !isAuthenticating && (
+          <div
+            className={classNames(
+              'flex flex-1 flex-col laptop:max-w-[27.5rem] laptop:mr-8',
+            )}
+          >
+            <OnboardingTitle className="typo-large-title tablet:typo-mega1">
+              Where developers grow together
+            </OnboardingTitle>
+
+            <h2 className="mb-8 typo-body tablet:typo-title2">
+              Get one personalized feed for all the knowledge you need.
+            </h2>
+
+            <AuthOptions
+              trigger={AuthTriggers.Filter}
+              formRef={formRef}
+              simplified
+              defaultDisplay={AuthDisplay.OnboardingSignup}
+              onSuccessfulLogin={onSuccessfulTransaction}
+              onSuccessfulRegistration={onSuccessfulTransaction}
+              isLoginFlow={isLoginFlow}
+              className={classNames('w-full', maxAuthWidth)}
+              onAuthStateUpdate={(props) =>
+                setAuth({ isAuthenticating: true, ...props })
+              }
+            />
+
+            <div className="flex relative flex-col tablet:flex-row flex-1 justify-end tablet:items-center pb-6 tablet:pb-0 mt-8 tablet:mt-auto w-full tablet:max-h-[10rem]">
+              <SignupDisclaimer className="mb-auto tablet:mb-0" />
+
+              <div className="block tablet:hidden">
+                <TrustedCompanies iconSize={IconSize.Small} />
+
+                <p className="relative z-3 mb-6 text-center opacity-64 typo-callout text-theme-label-quaternary">
+                  Trusted by 300K+ developers from the world&apos;s leading
+                  companies
+                </p>
+              </div>
```

Comment:
Why do we duplicate this section?

Reply:
Just refactored it into the `TrustedCompanies` component.